pal/system-info/minigame/system-info.ts

```diff
@@ -243,9 +243,16 @@ class SystemInfo extends EventTarget {
         }
     }
 
-    public close (): void {
+    public exit (): void {
         minigame.exitMiniProgram?.();
     }
```

Comment:
please don't add new public interface for `pal/system-info` module

what's the difference between close and exit, can they merge into one interface?

Reply:
Yep, it is confused.

Reply:
It cannot be divided into two interfaces.
Both interfaces should be public.

There are several processes for destruction:
1. Triggering the start of destruction, which is the role of 'close';
2, Ts engine to destroy resources, that is, the need to send the message close;
3, After the engine is destroyed, the active call to exit the program.

Reply:
I think only engine internally need to care about these processes. From the view of developer, just need one interface. Is there any situation that developer should care about these processes?

Reply:
Yes, it is an internal engine interface. Is 'systemInfo' not exported to the user?

Reply:
@PPpro please check if `systemInfo` is exported to developers.

Reply:
no, we only export `sys` to developers, which is a wrapper of `systemInfo`

Reply:
Is it more meaningful to name `close()` as `clearMemory()` ?

Reply:
It doesn't feel right because it's a process that closes.
We can change 'exit' to 'exitInternal' or 'closeInternal'？

Reply:
Can merge them into on interface and do close and exit in this interface?

Reply:
We can do this, but there will be multiple implementations for different platforms

Reply:
Yep, can do it in another PR.